### ISSUE TITLE
 Hubspot CRM integration

### ISSUE THREAD

**Issue by jessenieminen** [mautic/mautic#781]
Hubspot CRM is an awesome free CRM. We and many other potential Mautic users are using it so it would be great automatically sync new contacts and events relating to them to Hubspot.

**Comment by dbhurley** [comment]
Great suggestion, it's a really nice CRM and would be a super integration.

**Comment by jessenieminen** [comment]
Thanks for the reply! I'm looking forward to the integration.

Would you have any estimate on when we could expect this integration?

I was trying to make do with using the BCC address for Hubspot CRM for syncing at least the email address of the contact to the CRM but it doesn't work due to Hubspot apparently considering the message to be "sent on behalf of"  (according to their support) my user, even though I've added DKIM, SPF and domain validation for emails sent from Mautic...

**Comment by gpassarelli** [comment]
I've started working on this integration, will make a PR when it's done.

**Comment by Robmobius** [comment]
Hi, I have created a github account for the sole purpose of getting some support with the hubspot mautic integration. I can't get it to work. I am using the allyde hosted version of mautic so I don't have access to the code or logs.  I have asked for assistance from Allyde and from the Mautic community but no joy.  I have set up the integration correctly in the I have generated a hubspot API and mapped fields. I have set a campaign that pushes a list to Hubspot. The campaign fires correctly, but no records ever arrive in hubspot. Can anyone assist?
